### ISSUE TITLE
Define or remove GTPSF

### ISSUE THREAD

**Issue by MaxNoe** [open-gamma-ray-astro/gamma-astro-data-formats#152]
Currently this: https://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/full_enclosure/psf/psf_gtpsf/index.html
is missing any explanation / definition in what should be stored there.

We should either provide a proper definition or remove it.

**Comment by lmohrmann** [comment]
I tend to agree, although I'm not sure whether this format is maybe used by someone. Anyone knows what it's supposed to mean?

**Comment by adonath** [comment]
@maxnoe and @lmohrmann This is the format the Fermi-LAT PSF is stored in. Typically it's the output of the `gtpsf` tool. In Gammapy we support this format e.g. with the `PSFMap` now. Strictly speaking I would say it's not DL3 but rather DL4 data level. It could be useful to store e.g. PSF for spectral analyses or non-spatially varying PSFs. However I think it's basically unmodified from the Fermi's originally format, so I would not see GADF to be responsible for its documentation....so +1 to remove.
